@xen-orchestra/fs/src/local.js

```diff
@@ -1,10 +1,18 @@
 import df from '@sindresorhus/df'
 import fs from 'fs-extra'
-import { fromEvent } from 'promise-toolbox'
+import { fromEvent, retry } from 'promise-toolbox'
 
 import RemoteHandlerAbstract from './abstract'
 
 export default class LocalHandler extends RemoteHandlerAbstract {
+  constructor(remote, { delay = 1e3, retries = 9, when = { code: 'EAGAIN' } } = {}) {
```

Comment:
`delay`, `retries` and `when` are too generic options for a remote handler.

Do like the example I linked you in the other comment, use a single option with a specific name like `retryOnEagain`.